set.mm

```diff
@@ -524358,6 +524388,26 @@ to being a member of a class abstraction (that every class is equal to a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Characterization among sets versus among classes
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( Equivalence between two common ways to characterize elements of a class
+     ` B ` : the LHS says that sets are elements of ` B ` if and only if they
+     satisfy ` ph ` while the RHS says that classes are elements of ` B ` if
+     and only if they are sets and satisfy ` ph ` .  Therefore, the LHS is a
+     characterization among sets while the RHS is a characterization among
+     classes.  Note that the LHS is often formulated using a class variable
+     instead of the universe ` _V ` while this is not possible for the RHS
+     (apart from using ` B ` itself, which would not be very useful).
+     (Contributed by NM, 26-Feb-2023.) $)
```

Comment:
Oops, the contribution date cannot be correct...

Reply:
bad copy-paste... will fix.